care/abdm/utils/fhir.py

```diff
@@ -304,7 +314,7 @@ def _observations_from_daily_round(self, daily_round):
             ),
             self._observation(
                 "SpO2",
-                {"value": daily_round.spo2, "unit": "%"},
+                {"value": daily_round.archived_spo2, "unit": "%"},
```

Comment:
```suggestion
                {"value": daily_round.ventilator_spo2, "unit": "%"},
```

we should use ventilator_spo2 in place of the unused field

cc: @khavinshankar

Reply:
Yes 👍